website/projects/map4sci/src/app/map/map.component.ts

```diff
@@ -1,18 +1,19 @@
 import { Any } from '@angular-ru/common/typings';
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FeatureCollection } from 'geojson';
-import { FullscreenControl, Map, MapLayerMouseEvent, Marker, NavigationControl, Style, Popup, Layout } from 'mapbox-gl';
+import { FullscreenControl, Layout, Map, MapLayerMouseEvent, Marker, NavigationControl, Popup, Style } from 'mapbox-gl';
 
-import { Cluster, Edge, MapMarker, MiniMapOptions, Node, PopupContent, ZoomLookup, PopupLayer } from './map';
+import { Edge, MapMarker, MiniMapOptions, Node, PopupContent, PopupLayer, ZoomLookup } from './map';
 import { MiniMap } from './minimap';
 import { ZoomLevelControl } from './zoom-level.control';
 
 
+const defaultInitialZoom = 2;
```

Comment:
Move all these default constants out to a separate map.defaults.ts. The component is getting a little unwieldy at this point.